tools/code_format/check_format.py

```diff
@@ -928,6 +937,9 @@ def checkFormat(file_path):
 
   if error_messages:
     return ["From %s" % file_path] + error_messages
+
+  if file_path.endswith(PROTO_SUFFIX): 
```

Comment:
Shouldn't this be under `checkSourcePath`? There is already an API block there.

Reply:
done.